pkg/cmd/roachtest/main_test.go

```diff
@@ -233,12 +420,73 @@ func getTestSelectionMockData() ([]registry.TestSpec, *sqlmock.Rows) {
 		{"t_randomized", "no", "5", "no"},
 	}
 	// rows are the rows that are returned by snowflake. The list of string represents the columns of each row
-	rows := sqlmock.NewRows([]string{
-		"name", "selected", "avg_duration", "last_failure_is_preempt",
-	})
+	rows := sqlmock.NewRows(testselector.AllRows)
 	// rows are populated with the data
 	for _, ds := range data {
 		rows.FromCSVString(strings.Join(ds, ","))
 	}
 	return specs, rows
 }
+
+const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+// generateRandomString generates a random string of a given length
+func generateRandomString(n int) (string, error) {
+	result := make([]byte, n)
+	for i := range result {
+		num, err := rand.Int(rand.Reader, big.NewInt(int64(len(charset))))
+		if err != nil {
+			return "", err
+		}
+		result[i] = charset[num.Int64()]
+	}
+	return string(result), nil
+}
+
+// generateRandomLength generates a random length between min and max
+func generateRandomLength(min, max int) (int, error) {
```

Comment:
`randutil` has a function for this:

`randutil.RandIntInRange()` and `randutil.RandInt63InRange()`